### PR TITLE
[Documentation]: update example usage for `azurerm_cdn_frontdoor_custom_domain` and `azurerm_cdn_frontdoor_custom_domain_association`

### PR DESCRIPTION
**Pull request by WodansSon**
Update examples to be valid HCL.

### CODE REVIEW

File: website/docs/r/cdn_frontdoor_custom_domain_association.html.markdown [hunk 1]
```diff
@@ -13,9 +13,100 @@ Manages the association between a Front Door (standard/premium) Custom Domain an
...
+  name                     = "example-endpoint"
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.example.id
+}
+
+resource "azurerm_cdn_frontdoor_rule_set" "example" {
+  name                     = "ExampleRuleSet"
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.example.id
+}
+
+resource "azurerm_cdn_frontdoor_route" "contoso" {
```
Comment (jackofallops) [inline website/docs/r/cdn_frontdoor_custom_domain_association.html.markdown:78]:
```suggestion
resource "azurerm_cdn_frontdoor_route" "example" {
```

File: website/docs/r/cdn_frontdoor_custom_domain_association.html.markdown [hunk 2]
```diff
@@ -13,9 +13,100 @@ Manages the association between a Front Door (standard/premium) Custom Domain an
...
+  forwarding_protocol    = "HttpsOnly"
+  https_redirect_enabled = true
+  patterns_to_match      = ["/*"]
+  supported_protocols    = ["Http", "Https"]
+
+  cdn_frontdoor_custom_domain_ids = [azurerm_cdn_frontdoor_custom_domain.contoso.id]
+  link_to_default_domain          = false
+}
+
+resource "azurerm_cdn_frontdoor_custom_domain" "contoso" {
```
Comment (jackofallops) [inline website/docs/r/cdn_frontdoor_custom_domain_association.html.markdown:95]:
```suggestion
resource "azurerm_cdn_frontdoor_custom_domain" "example" {
```

**Review by jackofallops — CHANGES_REQUESTED** [review]
Thanks @WodansSon - just 2 small changes, then this should be good to go.

File: website/docs/r/cdn_frontdoor_custom_domain_association.html.markdown (hunk 1 above)
Comment (WodansSon) [reply website/docs/r/cdn_frontdoor_custom_domain_association.html.markdown]: I did that intentionally to show the relationship between the `azurerm_cdn_frontdoor_route`, `azurerm_cdn_frontdoor_custom_domain` and the `azurerm_cdn_frontdoor_custom_domain_association` resources. That said I am happy to switch that around to make the documentation consistent. 🙂

File: website/docs/r/cdn_frontdoor_custom_domain_association.html.markdown (hunk 2 above)
Comment (WodansSon) [reply website/docs/r/cdn_frontdoor_custom_domain_association.html.markdown]: Same as above, I did that intentionally to show the relationship between the `azurerm_cdn_frontdoor_route`, `azurerm_cdn_frontdoor_custom_domain` and the `azurerm_cdn_frontdoor_custom_domain_association` resources. That said I am happy to switch that around to make the documentation consistent. 🙂

**Review by jackofallops — APPROVED** [review]
Thanks @WodansSon - LGTM now 👍

**Pull request merged**